src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -3272,90 +3273,110 @@ public DynamicJsonValue GetLogDetails(ClusterOperationContext context, int max =
             return json;
         }
 
-        public static IEnumerable<Client.ServerWide.Operations.MountPointUsage> GetMountPointUsageDetailsFor(StorageEnvironmentWithType environment, bool includeTempBuffers)
+        public IEnumerable<Client.ServerWide.Operations.MountPointUsage> GetMountPointUsageDetailsFor(StorageEnvironmentWithType environment, bool includeTempBuffers)
         {
             var fullPath = environment?.Environment.Options.BasePath.FullPath;
             if (fullPath == null)
                 yield break;
 
+            var sizeOnDisk = environment.Environment.GenerateSizeReport(includeTempBuffers);
```

Comment:
do we need to move it here?

Reply:
Can you explain?

Reply:
Paweł meant that `sizeOnDisk is not required until line 3286 where it was placed originally.

Reply:
Ping

Reply:
done